launch/home/etc/services.cfg

```diff
@@ -20,8 +20,8 @@ org.openhab.autoapprove:enabled=false
 org.eclipse.smarthome.binding.sonos:opmlPartnerID=IAeIhU42
 
 org.jupnp:multicastResponsePort=0
-org.jupnp:streamListenPort=8080
-org.jupnp:threadPoolSize=200
+org.jupnp:asyncThreadPoolSize=30
+org.jupnp:threadPoolSize=15
 
```

Comment:
I'm not an expert on how upnp works, but 45 threads in total, isn't that a bit much?